src/Umbraco.Examine.Lucene/ExamineLuceneFinalComposer.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Umbraco.Core;
+using Umbraco.Core.Composing;
+
+namespace Umbraco.Examine
+{
+    // examine's Lucene final composer composes after all user composers
+    // and *also* after ICoreComposer (in case IUserComposer is disabled)
+    [ComposeAfter(typeof(IUserComposer))]
+    [ComposeAfter(typeof(ICoreComposer))]
```

Comment:
This is the same process we use for other composers tha tneed to happen last - we do this in WebFinalComposer too which has the same comments "(in case IUserComposer is disabled)"